random_food_spawn_decay_model.py

ArranBolger opened the pull request.

Kattelijn committed:
Adding food decay rate

kaleem-01 committed:
Data collector setup

gasperkreft committed:
Birth and death

kaleem-01 committed:
added consumption rate

kaleem-01 committed:
merge

kaleem-01 committed:
merge

gasperkreft committed:
merge conflict

gasperkreft committed:
Merge branch 'master' of https://github.com/kaleem-01/ants-mesa

kaleem-01 committed:
conflictsssss

Kattelijn committed:
Predator added

Kattelijn committed:
Predator added

kaleem-01 committed:
conflicttssssss

kaleem-01 committed:
conflicttssssss

Kattelijn committed:
Predator issue fixed

kaleem-01 committed:
random food locs

kaleem-01 committed:
Delete ants/scheduler.py

kaleem-01 committed:
distance to food added

Kattelijn committed:
Added meal_sizes for Predator agent

Kattelijn committed:
Merge branch 'master' of https://github.com/kaleem-01/ants-mesa

kaleem-01 committed:
run model for certain steps

kaleem-01 committed:
Set up default values from the config file in the server

kaleem-01 committed:
typo

kaleem-01 committed:
Food radius rather than exact location

mgoadric commented:
Hello @ArranBolger @Kattelijn @kaleem-01 @gasperkreft I'm excited to see your interest in the ant simulation I wrote for Mesa v 2.4! My goal is for this repository to be a near match for the original ant simulation from NetLogo. Can you tell me more about this pull request and your overall goals for the changes you propose? Thanks.

Kattelijn committed:
Added birth/death mechanism for Predator agent

kaleem-01 committed:
Parallelized simulations added

kaleem-01 committed:
parallel sims

kaleem-01 committed:
Merge branch 'master' of https://github.com/kaleem-01/ants-mesa
merge

gasperkreft committed:
configuration for the simulations

gasperkreft committed:
all predators

changed the model to keep track of all the predators, not just the ones that are alive

gasperkreft committed:
pheromone_count

kaleem-01 committed:
fixed git issue

kaleem-01 committed:
added gitignore again

kaleem-01 committed:
Merge branch 'master' of https://github.com/kaleem-01/ants-mesa

gasperkreft committed:
minor changes

gasperkreft committed:
Merge branch 'master' of https://github.com/kaleem-01/ants-mesa

gasperkreft committed:
merge

Kattelijn committed:
cleaning up

Kattelijn committed:
Fix path for notebooks

Kattelijn committed:
Delete empty file

mgoadric commented:
I'm going to close this, so I stop getting email updates from every commit made.

gasperkreft commented:
> Hello @ArranBolger @Kattelijn @kaleem-01 @gasperkreft I'm excited to see your interest in the ant simulation I wrote for Mesa v 2.4! My goal is for this repository to be a near match for the original ant simulation from NetLogo. Can you tell me more about this pull request and your overall goals for the changes you propose? Thanks.

Hello @mgoadric, we are a group of students at the University of Amsterdam. We are all following a course called Complex System Simulation and are trying to find some kind of emergent behaviour using your simulation for our project. We introduced a predator to the system and hope to see self-organised criticality in their interaction. Best, Gašper

mgoadric commented:
Thank you, that is helpful! I wish you the best of luck with your project!
